### ISSUE TITLE
Gulp build, dist folder still has all the TS files

### ISSUE THREAD

**Issue by mattiLeBlanc** [FountainJS/generator-fountain-webapp#95]
Hi,

I use fountain for a Angular, Typescript, Stylus project with PUG. Have that all working fine with help from fellow gitters.

I was evaluating the build, and I noticed that the TS files are still present in my dist folder.
I assume these need to be removed before the files are copied over to the dist folder since all the TS files have already been converted to JS.
Am I correct?

**Comment by kuhnroyal** [comment]
I changed a line in `gulp_tasks/misc.js`, added the `pug,ts`.

```
path.join(`!${conf.paths.src}`, '/**/*.{pug,ts,html,css,js,styl}')
```

**Comment by micaelmbagira** [comment]
Duplicate of https://github.com/FountainJS/generator-fountain-angular2/issues/70 but I'll leave it open because not the same repo.

**Comment by mattiLeBlanc** [comment]
@kuhnroyal Hi,

Yep, this works fine. Thanks

**Comment by micaelmbagira** [comment]
I reopen it until next release 😉

**Comment by kuhnroyal** [comment]
@mattiLeBlanc Since you use the pug-loader, you don't need the template-cache anymore. Try to remove all references to it.

**Comment by mattiLeBlanc** [comment]
@kuhnroyal Is that related to issue https://github.com/FountainJS/generator-fountain-webapp/issues/97 ?

**Comment by kuhnroyal** [comment]
No idea, I just know that you don't need it with pug-loader.

**Comment by micaelmbagira** [comment]
Fixed in 0.7.0 (just released) 😉

**Comment by mattiLeBlanc** [comment]
thanks guys! Great work.
